src/DataFixtures/ORM/LoadEventData.php

```diff
@@ -94,6 +94,7 @@ public function load(ObjectManager $manager)
             'begin_at' => (new Chronos('+3 days'))->format('Y-m-d').' 09:30:00',
             'finish_at' => (new Chronos('+3 days'))->format('Y-m-d').' 19:00:00',
             'capacity' => 10,
+            'time_zone' => 'Europe/Paris',
```

Comment:
Isn't `'Europe/Paris'` the default timezone ?

Reply:
the timeZone is not required for build Event